tests/admin_views/tests.py

```diff
@@ -4688,6 +4688,18 @@ def test_readonly_backwards_ref(self):
         response = self.client.get(reverse('admin:admin_views_topping_add'))
         self.assertEqual(response.status_code, 200)
 
+    @override_settings(DEBUG=True)
```

Comment:
Right.
However, without it, exception won't raise and `self.assertEqual(response.status_code, 200)` would always pass.